tests/test_notifications.py

```diff
@@ -401,16 +400,17 @@ def test_get_all_user_subscriptions(self):
         assert_in(self.project_subscription, user_subscriptions)
         assert_in(self.node_subscription, user_subscriptions)
         assert_in(self.user_subscription, user_subscriptions)
-        assert_equal(len(user_subscriptions), 3)
+        assert_equal(len(user_subscriptions), 5)
 
     def test_get_all_node_subscriptions_given_user_subscriptions(self):
         user_subscriptions = utils.get_all_user_subscriptions(self.user)
         node_subscriptions = [x for x in utils.get_all_node_subscriptions(self.user, self.node,
-                                                                          user_subscriptions=user_subscriptions)]
+                                                                          user_subscriptions=user_subscriptions)
+                                if x.event_name != 'mailing_list_events']
```

Comment:
This has been fixed by setting up the test class better. The one in [this test](https://github.com/CenterForOpenScience/osf.io/blob/8892077085897c0f6f74b64b3a14a5aaa15b14c1/tests/test_notifications.py#L685) is still there because ML subscriptions aren't inherited.